data/walkthroughs/scenarios/buyers/2.3/sidebar-content/9.tsx

```diff
@@ -0,0 +1,7 @@
+export const sidebarContent9 = (
+  <p>
+    The key lesson of this walkthrough is that how well a buyer fare's in the
+    market not only depends on how they bid but also on the level of competition
+    that they face in sourcing credits.
+  </p>
```

Comment:
Just noticed that this is (almost) a replication of the last paragraph in the step before. So I guess we go ahead and delete one of them.

I wonder if this one was introduced purely so we could have that return screen in the main PR comment 🤔